src/swatiShinde30/assignment3/Question1.java

```diff
@@ -0,0 +1,46 @@
+package swatiShinde30.assignment3;
+
+import java.util.Scanner;
+
+/*
+ * WAP which fulfill below condition 
+ * a. WAP to find positive and negative number from the given array  
+ * b. check the sum of individual. 
+ */
+public class Question1 {
+	static int positiveSum;
+	static int negativeSum;
+
+	public static void positiveNegativeNum(int[] input) {
+		for (int i = 0; i < input.length; i++) {
+			if (input[i] < 0) {
+				System.out.print(" " + input[i]);
+				negativeSum = negativeSum + (input[i]);
+				// System.out.println("\n");
+			} else {
+				positiveSum = positiveSum + (input[i]);
+			}
+		}
+	}
+	public static void main(String[] args) {
+		Scanner s = new Scanner(System.in);
+		System.out.println("Enter array size");
+		int size = s.nextInt();
+		int[] input = new int[size];
+		for (int i = 0; i < size; i++) {
```

Comment:
You have to store elements in another array not to print,
output is expected but need to understand requirement first.